@martenole : Sorry, but there are more errors in the checkcode now. Could you fix it:
```
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/4747/0/GPU/GPUTracking/DataCompression/GPUTPCCompressionTrackModel.h:128:15: error: field declaration 'kParamRMS0' does not match naming rule [aliceO2-member-name]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/4747/0/GPU/GPUTracking/DataCompression/GPUTPCCompressionTrackModel.cxx:906:21: error: field reference 'kParamRMS0' does not match naming rule [aliceO2-member-name]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/4747/0/GPU/GPUTracking/DataCompression/GPUTPCCompressionTrackModel.cxx:910:21: error: field reference 'kParamRMS0' does not match naming rule [aliceO2-member-name]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/4747/0/GPU/GPUTracking/DataCompression/GPUTPCCompressionTrackModel.h:128:15: error: field declaration 'kParamRMS0' does not match naming rule [aliceO2-member-name]
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/4747/0/GPU/GPUTracking/DataCompression/GPUTPCCompressionTrackModel.h:128:15: error: field declaration 'kParamRMS0' does not match naming rule [aliceO2-member-name]
```

--- a/GPU/GPUTracking/DataCompression/GPUTPCCompressionTrackModel.cxx
+++ b/GPU/GPUTracking/DataCompression/GPUTPCCompressionTrackModel.cxx
@@ -903,11 +903,11 @@ GPUd() void GPUTPCCompressionTrackModel::getClusterRMS2(int iRow, float z, float
   float angleY2 = s2 * sec2;          // dy/dx
   float angleZ2 = DzDs * DzDs * sec2; // dz/dx
 
-  const float* cY = kParamRMS0[0][rowType];
+  const float* cY = mParamRMS0[0][rowType];
   ErrY2 = cY[0] + cY[1] * z + cY[2] * angleY2;
   ErrY2 *= ErrY2;
 
-  const float* cZ = kParamRMS0[1][rowType];
+  const float* cZ = mParamRMS0[1][rowType];
   ErrZ2 = cZ[0] + cZ[1] * z + cZ[2] * angleZ2;
   ErrZ2 *= ErrZ2;
 }

--- a/GPU/GPUTracking/DataCompression/GPUTPCCompressionTrackModel.h
+++ b/GPU/GPUTracking/DataCompression/GPUTPCCompressionTrackModel.h
@@ -125,7 +125,7 @@ class GPUTPCCompressionTrackModel
 
   // default TPC cluster error parameterization taken from GPUParam.cxx
   // clang-format off
-  const float kParamRMS0[2][3][4] =
+  const float mParamRMS0[2][3][4] =
   {
     { { 4.17516864836e-02, 1.87623649254e-04, 5.63788712025e-02, 5.38373768330e-01, },
     { 8.29434990883e-02, 2.03291710932e-04, 6.81538805366e-02, 9.70965325832e-01, },

